dev/update_arrow_deps.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
```

Comment:
this probably isn't necessary but it also won't hurt - it was used when we had to sync arrow manually with datafusion before there were bi-weekly builds